plugins/importexport/native/filter/IssueNativeXmlFilter.inc.php

```diff
@@ -288,6 +281,39 @@ function addSections($doc, $issueNode, $issue) {
 
 		$issueNode->appendChild($sectionsNode);
 	}
+
+	/**
+	 * Create and return an issue identification node.
+	 * @param $doc DOMDocument
+	 * @param $issue Issue
+	 * @return DOMElement
+	 */
+	function createIssueIdentificationNode($doc, $issue) {
```

Comment:
Yes, do you maybe have an idea where to put this function then? It is used in two different filters, but logically it doesn't belong to their parent NativeImportExportFilter class, it is not so generic/common. Hmmm... :-)